stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,33 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute']
+};
+
+export const Text = () => `
+
+<div class="fd-object-attribute" id="standardtext" aria-label="Standard Object Attribute Text">Standard Object Attribute Text</div>
+`;
+
+Text.parameters = { docs: { disable: true } };
+
+
+export const TextWithLink = () => `
+<div id="firsttext" class="fd-object-attribute" aria-label="Object Attribute">Object Attribute :
+<span id="textwithlink" class="fd-object-attribute--link" aria-label="Text With Link" role="link" tabindex="0">Text With Link</span></div>
```

Comment:
1. Is there a reason this cannot instead be an `a`?
2. `aria-label` is redundant

```suggestion
<a href="https://link" id="textwithlink" class="fd-object-attribute--link">Text With Link</a></div>
```

Reply:
Hi @jacobdevera , this should be in a position to trigger any action on user demand, such as click open a dialog or popup, alert message, or navigation etc. because of this requirement, we are not restricting it as <a here.